lib/install/management/create.go

```diff
@@ -334,7 +334,7 @@ func (d *Dispatcher) cleanupAfterCreationFailed(conf *config.VirtualContainerHos
 	if d.isVC {
 
 		// we don't know if the appliance or the folder was made. so recreate the folder path.
-		vchFolder := fmt.Sprintf("%s/%s", d.session.VMFolder, conf.Name)
+		vchFolder := fmt.Sprintf("%s/%s", d.session.VMFolder.InventoryPath, conf.Name)
```

Comment:
Should we use `path.Join` here instead? There's no behavioral difference, but I think we prefer `path.Join` over `fmt.Sprintf` for constructing paths. FWIW, `path` is imported here as well.

Reply:
yeah I used that elsewhere before as well. It only applies to "slash based pathing" which should apply here. I was hesitant to use it originally.